imls-frontend/src/components/FetchData.vue

```diff
@@ -32,6 +32,17 @@ export default {
       isLoading: false,
     }
   },
+  computed: {
+    queryString() {
+      if (this.queryParams && Object.keys(this.queryParams).length !== 0) {
+        return '&' + Object.keys(this.queryParams).map(key => key + '=' + this.queryParams[key]).join('&');
```

Comment:
Line 38 here could use a comment explaining what's being constructed with this concatenation